modules/oe_multilingual_url_suffix/src/Event/UrlSuffixesAlterEvent.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_multilingual_url_suffix\Event;
+
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Dispatched by LanguageNegotiationUrlSuffix::getUrlSuffixes().
+ *
+ * Allows to alter the list of url suffixes retrieved from config.
```

Comment:
Allows the alter the list of URL suffixes available to the LanguageNegotiationUrlSuffix negotiator.